src/utils/statusUtil.ts

```diff
@@ -0,0 +1,19 @@
+import logger from './loggerUtil';
+import {PresenceData} from 'discord.js';
+
+var statuses = [
+    {status: 'online', activities: [{type: 'LISTENING', name: 'Martingales | $help'}]},
+    {status: 'online', activities: [{type: 'LISTENING', name: '115 ASMR | $help'}]},
+    {status: 'online', activities: [{type: 'WATCHING', name: 'Geese honk | $help'}]},
+    {status: 'online', activities: [{type: 'WATCHING', name: 'Crowdmark | $help'}]},
+    {status: 'online', activities: [{type: 'STREAMING', name: 'EGAD Videos | $help'}]},
+    {status: 'online', activities: [{type: 'PLAYING', name: 'Solidworks | $help'}]},
+    {status: 'online', activities: [{type: 'PLAYING', name: 'Among Us | $help'}]},
+]
+
+export const getRandomStatus = () => {
```

Comment:
I think you should actually be able to specify that this function returns PresenceData using : PresenceData
And you wont have to type 'as PresenceData' in the config. See if you can make this change. it didn't work earlier in testing because of the other issue we resolved.

Reply:
Didn't work